lib/Serialization/ModuleFormat.h

```diff
@@ -58,7 +58,8 @@ const uint16_t SWIFTMODULE_VERSION_MAJOR = 0;
 /// describe what change you made. The content of this comment isn't important;
 /// it just ensures a conflict if two people change the module format.
 /// Don't worry about adhering to the 80-column limit for this line.
-const uint16_t SWIFTMODULE_VERSION_MINOR = 958; // SILGlobalVariable markedAsUsed
+
+const uint16_t SWIFTMODULE_VERSION_MINOR = 959; // SILGlobalVariable markedAsUsed
```

Comment:
You need to update the comment as well with something like `// removed assign_by_wrapper instruction`